v1/lib/core/renderMarkdown.js

```diff
@@ -67,7 +68,14 @@ class MarkdownRenderer {
       },
       html: true,
       linkify: true,
-    });
+    };
+
+    // Allow client to use there own config
+    if (siteConfig.markdownConfig) {
+      markdownConfig = _.merge({}, markdownConfig, siteConfig.markdownConfig);
```

Comment:
We don't need to import whole lodash library just to use merge. In fact, we have deepmerge installed
https://github.com/facebook/Docusaurus/blob/3c36a1cc8cdfc5b83add6dcdf2239e26cb3c783e/v1/package.json#L46

```js
markdownConfig = deepmerge({}, markdownConfig, siteConfig.markdownConfig)
```